packs/st2/actions/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: "send_anonymous_install_data"
```

Comment:
Come on :) 
call it `callhome` and use `description: Send anonymous install data`. 
Note that it's not only "install" data, we will likely be using it elsewhere, like in "packs.install" or "packs.download" flows.